src/Npgsql/NpgsqlDataSourceBuilder.cs

```diff
@@ -325,6 +325,19 @@ public NpgsqlDataSourceBuilder UseRootCertificateCallback(Func<X509Certificate2>
         return this;
     }
 
+#if NET7_0_OR_GREATER
+    /// <summary>
+    /// When using Kerberos, this is a callback that allows customizing default settings for Kerberos authentication.
+    /// </summary>
+    /// <param name="negotiateOptionsCallback">The callback containing logic to customize Kerberos authentication settings.</param>
+    /// <returns>The same builder instance so that multiple calls can be chained.</returns>
```

Comment:
Link to the .NET API docs? https://learn.microsoft.com/en-us/dotnet/api/system.net.security.negotiateauthenticationclientoptions?view=net-7.0

Reply:
Yeah, good idea.

Reply:
Done